github/issues.go

```diff
@@ -97,7 +105,7 @@ type IssueListOptions struct {
 }
 
 // PullRequestLinks object is added to the Issue object when it's an issue included
-// in the IssueCommentEvent webhook payload, if the webhooks is fired by a comment on a PR
+// in the IssueCommentEvent webhook payload, if the webhook is fired by a comment on a PR
```

Comment:
nit: while you are here, can you please add a period to the end of this sentence?